libs/model/src/contest/Contests.projection.ts

```diff
@@ -101,7 +102,7 @@ async function updateOrCreateWithAlert(
     const { startTime, endTime } = await getContestStatus(
```

Comment:
are we still creating the contest here? or is this a temporary solution?

Reply:
Temp solution. The onchain contract needs to be updated to properly emit the event before the code can be refactored. IMO it's a higher-level discussion w/ protocol about the contract reflecting the model. This is simply one case of several where it doesn't match and we have to hack around it.

Reply:
Per discussion w/ Ian, this will be the official implementation:
- New deployment is the implicit start of both one-off and recurring contests
- Then the NewContest event fires for every subsequent interval of a recurring contest